runelite-client/src/main/java/net/runelite/client/game/AgilityShortcut.java

```diff
@@ -245,4 +245,9 @@ public String getTooltip()
 	{
 		return description + " - Level " + level;
 	}
+
+	public int getLevel()
```

Comment:
Why was this added? This is already created via Lombok.

Reply:
Whoops, I must have missed that.